src/applications/hca/reducers/hca-enrollment-status-reducer.js

```diff
@@ -62,9 +62,10 @@ function hcaEnrollmentStatus(state = initialState, action) {
       const { errors } = action;
       const noESRRecordFound =
         errors && errors.some(error => error.code === '404');
-      const hasServerError = errors && errors.some(error => error.code >= 500);
```

Comment:
I discovered that `hasServerError` is `false` when running Cypress tests and the `GET enrollment_status` is not mocked at all but it should be `true` in that scenario. So I'm reevaluating how this value is set. We really want it to be `true` whenever we encounter any error that we are not giving special handling to.